Manifests/ManagedPreferencesApplications/com.twocanoes.xcreds.plist

```diff
@@ -520,7 +520,71 @@ A profile can consist of payloads with different version numbers. For example, c
 			<key>pfm_name</key>
 			<string>passwordElementID</string>
 			<key>pfm_title</key>
-			<string>Password Element ID</string>
+			<string>passwordElementID</string>
+			<key>pfm_type</key>
+			<string>string</string>
+		</dict>
+		<dict>
+			<key>pfm_default</key>
+			<string>given_name</string>
+			<key>pfm_description</key>
+			<string>Local DS to OIDC Mapping for First Name</string>
+			<key>pfm_documentation_url</key>
+			<string>https://github.com/twocanoes/xcreds/wiki/AdminGuide#oidcmappingfirstname</string>
+			<key>pfm_name</key>
+			<string>map_firstname</string>
+			<key>pfm_note</key>
+			<string>Map firstName to OIDC claim</string>
+			<key>pfm_title</key>
+			<string>First Name OIDC Mapping</string>
+			<key>pfm_type</key>
+			<string>string</string>
+		</dict>
+		<dict>
+			<key>pfm_default</key>
+			<string>family_name</string>
+			<key>pfm_description</key>
+			<string>Local DS to OIDC Mapping for Last Name</string>
+			<key>pfm_documentation_url</key>
+			<string>https://github.com/twocanoes/xcreds/wiki/AdminGuide#oidcmappinglastname</string>
+			<key>pfm_name</key>
+			<string>map_lastname</string>
```

Comment:
Same as above, this should probably be `oidcmappinglastname`.

Reply:
See https://github.com/DigiDNA/ProfileManifests/pull/2#discussion_r1117045104